src/content/en/tools/workbox/guides/_shared/base-schema.html

```diff
@@ -0,0 +1,229 @@
+    <p>
+      For more information, see the definition of <code>ignore</code> in the <code>glob</code>
+      <a href="https://github.com/isaacs/node-glob#options" class="external">documentation</a>.
+    </p>
+    <p>
+      <strong>Example:</strong>
+    </p>
+    <pre class="prettyprint">globIgnores: ['**/ignored.html']</pre>
+  </td>
+</tr>
+
+<tr>
+  <td>
+    <p>globPatterns</p>
+  </td>
+  <td>
+    <p>
+      <em>Optional <code>Array</code> of <code>String</code>, defaulting to
+        <code>['**/*.{js,css,html}']</code></em>
+    </p>
+    <p>
+      Files matching against any of these patterns will be included in the precache manifest.
+    </p>
+    <p>
+      For more information, see the
+      <a href="https://github.com/isaacs/node-glob#glob-primer" class="external">glob primer</a>.
+    </p>
+    <p>
+      <strong>Example:</strong>
+    </p>
+    <pre class="prettyprint">globPatterns: ['dist/*.{js,png,html,css}']</pre>
+  </td>
+</tr>
+
+<tr>
+  <td>
+    <p>globStrict</p>
+  </td>
+  <td>
+    <p>
+      <em>Optional <code>Boolean</code>, defaulting to <code>true</code></em>
+    </p>
+    <p>
+      If <code>true</code>, an error reading a directory when generating a precache manifest will
+      cause the build to fail. If <code>false</code>, the problematic directory will be skipped.
+    </p>
+    <p>
+      For more information, see the definition of <code>strict</code> in the <code>glob</code>
+      <a href="https://github.com/isaacs/node-glob#options" class="external">documentation</a>.
+    </p>
+    <p>
+      <strong>Example:</strong>
+    </p>
+    <pre class="prettyprint">globStrict: false</pre>
+  </td>
+</tr>
+
+<tr>
+  <td>
+    <p>manifestTransforms</p>
+  </td>
+  <td>
+    <p>
+      <em>Optional <code>Array</code> of <code>ManifestTransform</code>, defaulting to
+        <code>null</code></em>
+    </p>
+    <p>
+      One or more
+      <a href="/web/tools/workbox/reference-docs/prerelease/module-workbox-build.html#.ManifestTransform"><code>ManifestTransform</code></a>
+      functions, which will be applied sequentially against the generated manifest.
+    </p>
+    <p>
+      If <code>modifyUrlPrefix</code> or <code>dontCacheBustUrlsMatching</code>
+      are also specified, their corresponding transformations will be applied first.
+    </p>
+    <p>
+      <strong>Example:</strong>
+    </p>
+    <pre class="prettyprint">manifestTransforms: [
+  // Basic transformation to ignore a certain URL:
+  (entries) => entries.filter((entry) => entry.url !== 'ignored.html')
+]</pre>
+  </td>
+</tr>
+
+<tr>
+  <td>
+    <p>maximumFileSizeToCacheInBytes</p>
+  </td>
+  <td>
+    <p>
+      <em>Optional <code>Number</code>, defaulting to <code>2097152</code></em>
+    </p>
+    <p>
+      This value can be used to determine the maximum size of files that will be precached. This
+      prevents you from inadvertantly precaching very large files that might have been accidentally
+      match one of your patterns.
```

Comment:
Nit: "might have accidentally matched one of your patterns."

Reply:
Done.